backend/src/main/java/de/kathahuth/gymast/model/Group.java

```diff
@@ -9,9 +10,12 @@
 @Data
 @AllArgsConstructor
 @NoArgsConstructor
+@Builder
 @Document(collection = "groups")
 public class Group {
 
     @Id
-    private String name;
+    private String id;
```

Comment:
you can also use name as `@Id` attribute name, if you are using a name I would recommend using `name` instead